mesonbuild/backend/ninjabackend.py

```diff
@@ -2303,6 +2311,17 @@ def generate_dynamic_link_rules(self):
 
                 options = self._rsp_options(compiler)
                 self.add_rule(NinjaRule(rule, command, args, description, **options, extra=pool))
+            if mesonlib.is_aix(): 
+                rule = 'AIX_LINKER{}'.format(self.get_rule_suffix(for_machine))
+                #Archive the shared library and remove the shared library since it is already there in the archive.
+                cmdlist = ['ar']
+                cmdlist += ['-q', '-v']
+                #Remove the shared library since it is already there in the archive.
+                description = 'Archiving AIX shared library'
+                cmdlist+=['$out', '$in', '&&', 'rm', '-f', '$in']
```

Comment:
Space around operator.

Reply:
This is done. Kindly check the latest commit